client/src/components/viewToolbar/viewToolbar.js

```diff
@@ -218,7 +218,7 @@ class ViewToolbar extends React.Component {
 
     return (
       <h5>
-        {activeFilters && activeFilters.length > 0
+        {filteredCount >= 0 && activeFilters && activeFilters.length > 0
           ? `${filteredCount} of `
           : null}
         {totalCount + ' ' + (totalCount === 1 ? itemsType : itemsTypePlural)}
```

Comment:
Optional: heh, may have commented on this before, and it is existing, a string/template literal for ```totalCount``` could work